PWGJE/EMCALJetTasks/AliEmcalJetTask.cxx

```diff
@@ -1057,6 +1066,50 @@ void AliEmcalJetTask::LoadTrackEfficiencyFunction(const std::string & path, cons
   delete file;
 }
 
+/**
+ * Set the pt-dependent tracking efficiency depending on periodfrom a YAML file
+ * @param path Path to the file containing the TF1
+ * @param name Name of the TF1
+ */
+
+void AliEmcalJetTask::SetTrackingEfficiencyFromYAML(std::string period , std::string path ) {
+
+  AliInfoStream() << "Get pT-dependent Tracking efficiency from" << path << "\".\n";
+  
+  int addedConfig = fYAMLConfig.AddConfiguration(path, "yamlConfig");
+  if (addedConfig < 0) {
+    AliFatal(Form("YAML Configuration in set path %s not found!",path.c_str()));
+  }
+
+  std::vector <Double_t> ptBinning;
+  std::vector <Double_t> trackingUncertainty;
+  bool res = fYAMLConfig.GetProperty("ptBinning", ptBinning, false);
+  Int_t nPtBins = ptBinning.size()-1;
+  double* aptBinning = &ptBinning[0];
+
+  TString sPeriod(period);
+
+  if(sPeriod=="LHC18q_010")       res = fYAMLConfig.GetProperty({"leadLead","LHC18q","central"},trackingUncertainty, false);
+  else if(sPeriod=="LHC18q_3050") res = fYAMLConfig.GetProperty({"leadLead","LHC18q","semiCentral"},trackingUncertainty, false);
+  else if(sPeriod=="LHC18r_010")  res = fYAMLConfig.GetProperty({"leadLead","LHC18r","central"},trackingUncertainty, false);
+  else if(sPeriod=="LHC18r_3050") res = fYAMLConfig.GetProperty({"leadLead","LHC18r","semiCentral"},trackingUncertainty, false);
+  else {
+    AliFatal("Setting tracking efficiency period but the period selected is not implemented at present"); 
```

Comment:
Would be good to include the passed value to help folks out

Reply:
done